Exoft.Gamification.Api.Services/FileService.cs

```diff
@@ -30,5 +32,29 @@ public async Task<FileModel> GetFileByIdAsync(Guid id)
 
             return _mapper.Map<FileModel>(file);
         }
+
+        public async Task<Guid?> AddOrUpdateFileByIdAsync(IFormFile image, Guid? IconId)
```

Comment:
Maybe we need use SaveChanges in this method. What do you think?

Reply:
It is now used only in services, which have SaveChanges in the end. But I agree, we have to add method in service, it can be used anywhere else